rwengine/src/loaders/LoaderIFP.hpp

```diff
@@ -16,6 +16,17 @@ struct AnimationKeyframe {
     glm::vec3 scale;
     float starttime;
     int id;
+
+    AnimationKeyframe(glm::quat _rotation, glm::vec3 _position, glm::vec3 _scale, float _starttime, int _id)
+        : rotation(_rotation)
+        , position(_position)
+        , scale(_scale)
+        , starttime(_starttime)
+        , id(_id) {
+    }
+
+    AnimationKeyframe() {
```

Comment:
Is it safe to leave the members uninitialised? Where is this ctor used

Reply:
Here:
https://github.com/ShFil119/openrw/blob/0b11659c81b4ee3e8dcd813a84af6a4ff5bb3a7d/rwengine/src/loaders/LoaderIFP.cpp#L35
But few moment later it's overwritten here:
https://github.com/ShFil119/openrw/blob/0b11659c81b4ee3e8dcd813a84af6a4ff5bb3a7d/rwengine/src/loaders/LoaderIFP.cpp#L9
If you want, I can change.